app/service/document.py

```diff
@@ -136,3 +136,18 @@ def delete(import_id: str, db: Session = db_session.get_db()) -> bool:
     """
     id.validate(import_id)
     return document_repo.delete(db, import_id)
+
+
+@validate_call(config=ConfigDict(arbitrary_types_allowed=True))
```

Comment:
Looks like the analytics service uses the repo directly, so is there a need/use of this function?

Reply:
Updated analytics service to use the services instead of repositories.